lib/project_types/extension/features/argo.rb

```diff
@@ -23,17 +23,23 @@ def create(directory_name, identifier, context)
         Features::ArgoSetup.new(git_template: git_template).call(directory_name, identifier, context)
       end
 
-      def config(context)
+      def config(context, include_renderer_version = true)
```

Comment:
What exactly is `context`? Does it make sense for the `include_renderer_version` flag to be a part of the context?

Reply:
👍 Thanks for the clarification.